ang/civicase/case/directives/case-overview.directive.js

```diff
@@ -8,6 +8,7 @@
       templateUrl: '~/civicase/case/directives/case-overview.directive.html',
       controller: civicaseCaseOverviewController,
       scope: {
+        currentCaseCategory: '=',
```

Comment:
this value doesn't need to be injected to the directive as a scope parameter. it can, and should be injected as a service.

ie:

```js
function controller (currentCaseCategory) {

}
```